packages/ui-extensions/src/surfaces/admin/api/block/block.ts

```diff
@@ -23,4 +24,9 @@ export interface BlockExtensionApi<ExtensionTarget extends AnyExtensionTarget>
    * For example, you can navigate from an admin block on the product details page (`admin.product-details.block.render`) to an admin action on the product details page (`admin.product-details.action.render`).
    */
   navigation: Navigation;
+
+  /**
+   * Renders the [Resource Picker](https://shopify.dev/docs/api/app-bridge-library/apis/resource-picker), allowing users to select a resource for the extension to use as part of it's flow.
```

Comment:
Should be relative I think or it won't work in spin

```suggestion
   * Renders the [Resource Picker](/docs/api/app-bridge-library/apis/resource-picker), allowing users to select a resource for the extension to use as part of it's flow.
```